hyperopt/tests/test_spark.py

```diff
@@ -504,14 +504,6 @@ def fn(x):
                     log_output=log_output
                 ),
             )
-            self.assertNotIn(
-                "Task should have been cancelled",
-                log_output,
-                """ "Task should have been cancelled" should not in log:
-                              {log_output}""".format(
-                    log_output=log_output
-                ),
-            )
```

Comment:
This line test is meaningless and it will fail under spark 3.2, so I remove it.

The log will print a line like:
```
trial task {tid} cancelled, exception is {e}
```

The exception we get from spark 3.2 is the original exception `Exception("Task should have been cancelled")`, but in spark < 3.2 it will get an wrapped exception which make the log does not contains message "Task should have been cancelled", this difference make the line test fail under spark 3.2.
But, this line test is meaningless, we only need to test the job was cancelled correctly, so I remove this line.